src/screens/Login/index.test.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import configureMockStore from 'redux-mock-store';
+import { mount } from 'enzyme';
+import Login from './index';
+
+const mockStore = configureMockStore();
+const store = mockStore({
+  authenticate: {}
+});
+
+describe('Login component', () => {
+  it('renders without crashing', () => {
+    const wrapper = mount(
+      <Login store={store} />
+    );
+    expect(wrapper).toMatchSnapshot();
+  });
+
+
+  it('should handleChange function in email fields called', () => {
+    const wrapper = mount(
+      <Login store={store} />
+    );
+
+    const component = wrapper.find('#email');
+    component.simulate("change", {
+      target: { value: 'test content' },
+    });
```

Comment:
What's your expectation in this one and 2 more below? I think that we should expect the test does something.